CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [Unreleased]
```

Comment:
chagenlog should be automatically managed by goreleaser - no manual editing

Reply:
Reverted the CHANGELOG